.github/workflows/e2e-test.yml

```diff
@@ -1,15 +1,17 @@
 name: E2E Test
 
 on:
+#  workflow_dispatch:
+
+  pull_request:
+    branches:
+      - main
+
   workflow_run:
-    workflows: [Lint, Unit tests]
+    workflows: ["Lint", "Unit tests"]
     types: [completed]
     branches: [stage, main]
-  #  pull_request:
-  #    branches:
-  #      - "**"
 
-  workflow_dispatch:
```

Comment:
bring this back

Reply:
it's been done on the first PR

Reply:
so why is this part of this PR? please clear the commits